### PR TITLE
Added new crawler netaporter with 172 regions

### PR DESCRIPTION
**Pull request by saad-waqar**
172 regions for netaporter added witht the following names:
netaporter-al
netaporter-dz
netaporter-ad
netaporter-ai
netaporter-ag
netaporter-ar
netaporter-am
netaporter-aw
netaporter-au
netaporter-at
netaporter-az
netaporter-bs
netaporter-bh
netaporter-bd
netaporter-bb
netaporter-by
netaporter-be
netaporter-bz
netaporter-bm
netaporter-bt
netaporter-bo
netaporter-ba
netaporter-bw
netaporter-br
netaporter-vg
netaporter-bn
netaporter-bg
netaporter-kh
netaporter-ca
netaporter-cv
netaporter-ky
netaporter-cl
netaporter-cn
netaporter-co
netaporter-km
netaporter-ck
netaporter-cr
netaporter-hr
netaporter-cy
netaporter-cz
netaporter-dk
netaporter-dm
netaporter-do
netaporter-ec
netaporter-eg
netaporter-sv
netaporter-ee
netaporter-fk
netaporter-fo
netaporter-fj
netaporter-fi
netaporter-fr
netaporter-gf
netaporter-pf
netaporter-ga
netaporter-gm
netaporter-ge
netaporter-de
netaporter-gh
netaporter-gi
netaporter-gr
netaporter-gl
netaporter-gd
netaporter-gp
netaporter-gu
netaporter-gt
netaporter-gy
netaporter-hn
netaporter-hk
netaporter-hu
netaporter-is
netaporter-in
netaporter-id
netaporter-ie
netaporter-il
netaporter-it
netaporter-jm
netaporter-jp
netaporter-jo
netaporter-kz
netaporter-ke
netaporter-kw
netaporter-la
netaporter-lv
netaporter-lb
netaporter-ls
netaporter-li
netaporter-lt
netaporter-lu
netaporter-mo
netaporter-mk
netaporter-mg
netaporter-mw
netaporter-my
netaporter-mv
netaporter-mt
netaporter-mq
netaporter-mu
netaporter-mx
netaporter-md
netaporter-mc
netaporter-mn
netaporter-me
netaporter-ms
netaporter-ma
netaporter-mz
netaporter-na
netaporter-np
netaporter-nl
netaporter-an
netaporter-nc
netaporter-nz
netaporter-ni
netaporter-no
netaporter-om
netaporter-pa
netaporter-pg
netaporter-py
netaporter-pe
netaporter-ph
netaporter-pl
netaporter-pt
netaporter-pr
netaporter-qa
netaporter-ro
netaporter-ru
netaporter-kn
netaporter-lc
netaporter-vc
netaporter-ws
netaporter-sm
netaporter-st
netaporter-sa
netaporter-sn
netaporter-rs
netaporter-sc
netaporter-sg
netaporter-sk
netaporter-si
netaporter-sb
netaporter-za
netaporter-kr
netaporter-es
netaporter-ic
netaporter-lk
netaporter-bl
netaporter-sr
netaporter-sz
netaporter-se
netaporter-ch
netaporter-tw
netaporter-tz
netaporter-th
netaporter-tl
netaporter-tg
netaporter-to
netaporter-tt
netaporter-tn
netaporter-tr
netaporter-tc
netaporter-tv
netaporter-vi
netaporter-ua
netaporter-ae
netaporter-gb
netaporter-gg
netaporter-je
netaporter-us
netaporter-uy
netaporter-vu
netaporter-ve
netaporter-vn

### CODE REVIEW

File: fashion_stalkers/spiders/netaporter.py [hunk 1]
```diff
@@ -0,0 +1,1154 @@
...
+            sku['size'] = raw_sku['displaySize']
+            sku['measurements'] = {m['name']: m['value'] for m in raw_sku['measurements']}
+            sku['availability'] = raw_sku['stockLevel']
+
+            skus[raw_sku['id']] = sku
+
+        return skus
+
+
+
```
Comment (sanealytics) [inline fashion_stalkers/spiders/netaporter.py]: Can you factor the rest into another file please?
Also, do we raise an alarm if a new country shows up?

**Comment by sanealytics** [comment]
Also, would love to see how to tell women from men clothes. I don't see it in the categories data. Also, color is missing.
sizes seems to be an array of a dictionary... just make it a dictionary

**Comment by saad-waqar** [comment]
> Also, would love to see how to tell women from men clothes. I don't see it in the categories data. Also, color is missing.
> sizes seems to be an array of a dictionary... just make it a dictionary

Colours aren't explicitly given on net a porter. products with colours are listed separately(a different product altogether)
Net a porter only has women products. I couldn't find a men section. Should I add a default gender of women to net-a-porter?

**Comment by virevol** [comment]
Thank you. In that case, no need to add color or gender.
ETL will take care of it.
And I like the new file structure... much prettier.
I will approve these. Feel free to merge when you are ready (do not squash). 
I will go and pay the first two milestones on upwork.

Thanks for the good work.

**Pull request merged**